### DISCUSSION TITLE
Chia start commands do not start anything via nodejs.

### DISCUSSION THREAD

**Discussion by CameronAP** [Chia-Network/chia-blockchain#7163] [Support]
I am having a problem with a small monitoring program and I cannot get exec or execSync,  to execute the any of the start commands. I am not met with an error message it just doesnt progress further and gets stuck. Nothing comes up in the debug logs either.
This is the code im using to try and test it:

![image](https://user-images.githubusercontent.com/86672234/124758348-61f30580-df26-11eb-9f79-479251640794.png)

I am very new to nodejs and chia so there may be something really simple here.

Thanks.

**Comment by BrandtH22** [comment]
Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
